include/seqan3/alphabet/gap/all.hpp

```diff
@@ -0,0 +1,57 @@
+//                 SeqAn - The Library for Sequence Analysis
+// ============================================================================
+//
+// Copyright (c) 2006-2017, Knut Reinert & Freie Universitaet Berlin
+// Copyright (c) 2016-2017, Knut Reinert & MPI Molekulare Genetik
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+
+/*!\file
+ * \ingroup gap
+ * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
+ * \brief Meta-header for the gap submodule; includes all headers from alphabet/gap/.
+ * \group gap
+ */
+
+#include <seqan3/alphabet/gap/gap.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
+
+/*!\defgroup gap
+ * \brief Contains the gap alphabet and functionality to make an alphabet a gapped alphabet.
+ * \ingroup alphabet
+ *
+ * \par Introduction
+ * The gap '-' symbol is used in alignments to show that there is at a certain position an insertion or deletion between
+ * two or more input sequences. The seqan3::gap alphabet represents this (single) gap symbol and satisfies the
+ * seqan3::alphabet_concept.
+ *
+ * This enables us to combine any given alphabet with the seqan3::gap alphabet. This can be achieved by using the
+ * seqan3::gapped class, there you can transform any arbitrary alphabet into a gapped alphabet with the same symbols,
+ * but with the additional gap '-' symbol.
```

Comment:
I would suggest:
"The main purpose of seqan3::gap is to be combined with other alphabets. This can easily be achieved by using the seqan3::gapped<> template which transforms any other alphabet to be a composition of that alphabet + the gap character."